brands/itel/README.md

```diff
@@ -1,12 +1,17 @@
 # itel 
 
-* Verdict **ℹ️ "Safe for now" :trollface:**
+- Verdict: **⚠️ Proceed with caution!**
+
 * [**🔓️ Unlock Guide (MediaTek)**](/misc/generic-unlock.md)
 * [**🔓️ Unlock Guide (Unisoc)**][Unisoc Unlock]
 
+Itel requires you to have a Itel ID for two weeks before the OEM Unlocking option becomes available in the settings. Afterwards, you can follow the [standard unlock guide](/misc/generic-unlock.md) for devices with a MediaTek SoC, or the [Unisoc unlock guide][Unisoc Unlock] for models with a Unisoc SoC.
+
+Earlier, itel didn't require online verification and two weeks of waiting. Things changed.
+
+Also see Tecno and Realme - all these brands are Transsion corp (Chinese, yes)
```

Comment:
> Realme

Realme is an Oppo subsidiary.

Reply:
Yes, you are right, Realme is spin-off of Oppo
https://en.wikipedia.org/wiki/Realme

But, still, Realme, Oppo, Tecno, Infinix are all the Trassion corp.
https://en.wikipedia.org/wiki/Transsion

From technical point of view, the Tecno ID, locked bootloader, 2 weeks waiting and Itel ID, locked bootloader seems very similar

Reply:
No, Oppo was a subsidiary of BBK, but is independent since 2023. Transsion Holdings has nothing to do with Oppo.